src/pymedphys/dicom/_level2/dose.py

```diff
@@ -358,11 +358,11 @@ def extract_dicom_patient_xyz(ds):
 
     References
     ----------
-    .. [1] "C.7.6.2.1.1 Image Position and Image Orientation",
+    .. [5] "C.7.6.2.1.1 Image Position and Image Orientation",
        "DICOM PS3.3 2016a - Information Object Definitions",
        http://dicom.nema.org/MEDICAL/dicom/2016a/output/chtml/part03/sect_C.7.6.2.html#sect_C.7.6.2.1.1
 
-    .. [2] O. McNoleg, "Generalized coordinate transformations for Monte Carlo
+    .. [6] O. McNoleg, "Generalized coordinate transformations for Monte Carlo
```

Comment:
@Centrus007 By swapping to `sphinxcontrib.napoleon` it complained of duplicate references in `dose.py`. So I changed that. Let me know if you'd like it a little different.

Cheers,
Simon